### PR TITLE
225 credential logging integration test

### PR DESCRIPTION
**Pull request by l-friebe**

### CODE REVIEW

**Comment by cloudgge** [comment]
LGTM. 

But, any idea why the trivy scan passed in two executions?

I sadly cannot at the moment due to bugs with our trivy understand if we added a new vulnerability in the pipeline with this?

**Comment by l-friebe** [comment]
No I don't understand it either. [Commit 8d055c0](https://github.com/cloudogu/cas/pull/227/commits/8d055c07bf0063b4085ef54771b52be6e23efe1d) is the first one that fails again, but there are no real differences to the commit before.
For some reason I cannot access the build pipeline anymore.. So I can't say what Trivy complains about

**Comment by cloudgge** [comment]
All good. Ignore my comment. The trivy scan just seems to randomly fail.

One thing the log says: 
[2024-10-29T14:42:18.806Z] grep: /dogu/cas_internal_logs/cas.log: Permission denied

[2024-10-29T14:42:18.806Z] grep: /dogu/cas_internal_logs/cas_stacktrace.log: Permission denied

[2024-10-29T14:42:18.806Z] grep: /dogu/cas_internal_logs/cas_audit.log: Permission denied

Try sudo in your scripts.

**Comment by l-friebe** [comment]
Added sudo, manual pipeline already went trough without errors

File: Jenkinsfile [hunk 1]
```diff
@@ -196,15 +196,18 @@ parallel(
...
+                           ecoSystem.vagrant.sshOut "etcdctl set /dogu/inttest/current \"0.0.1\""
 
-                            ecoSystem.runCypressIntegrationTests([
+                           ecoSystem.runCypressIntegrationTests([
                                     cypressImage     : "cypress/included:13.13.2",
                                     enableVideo      : params.EnableVideoRecording,
-                                    enableScreenshots: params.EnableScreenshotRecording])
+                                   enableScreenshots: params.EnableScreenshotRecording])
+                           // run special non-encrypted password test
+                           ecoSystem.vagrant.sshOut 'chmod +x /dogu/resources/test-password-logging.sh'
```
Comment (cloudgge) [inline Jenkinsfile:209]: Let's add an echo before this. To signify what we are doing.

File: resources/test-password-logging.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
```
Comment (cloudgge) [inline resources/test-password-logging.sh:5]: Could use some echoes here and there.

Some Steps in between
Password (not expected) vs actual

Comment (cloudgge) [inline resources/test-password-logging.sh:5]: what about "set -e" as a failsafe so we dont miss an error again? Afaik it should also stop the script when anything errors out unexpectedly.

**Pull request merged**